admin/widgets.rb

```diff
@@ -0,0 +1,171 @@
+ActiveAdmin.register Goldencobra::Widget, as: "Widget" do
+  menu parent: I18n.t('active_admin.widget.parent'), label: I18n.t('active_admin.widget.as'), if: proc{can?(:update, Goldencobra::Widget)}
+
+  filter :title, label: I18n.t('active_admin.widget.title')
+  filter :css_name, label: I18n.t('active_admin.widget.css_class')
+  filter :id_name, label: I18n.t('active_admin.widget.id')
+  filter :sorter, label: I18n.t('active_admin.widget.sorter')
+
+  scope I18n.t('active_admin.widget.scope1'), :all, default: true
+  scope I18n.t('active_admin.widget.scope2'), :active
+  scope I18n.t('active_admin.widget.scope3'), :inactive
+  scope I18n.t('active_admin.widget.scope4'), :default
+
+  if ActiveRecord::Base.connection.table_exists?("tags")
+    Goldencobra::Widget.tag_counts_on(:tags).map(&:name).each do |wtag|
+      scope(I18n.t(wtag, scope: [:goldencobra, :widget_types], default: wtag).capitalize){ |t| t.tagged_with(wtag) }
+    end
+  end
+
+  form html: { enctype: "multipart/form-data" } do |f|
+    f.actions
+    f.inputs I18n.t('active_admin.widget.general'), class: "foldable inputs" do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.